ext/drwrap/drwrap.c

```diff
@@ -1008,6 +1008,12 @@ drwrap_exit(void)
         postcall_cache[i] = NULL;
     }
     postcall_cache_idx = 0;
+#ifdef WINDOWS
+    sysnum_NtContinue = -1;
+#endif
+    disabled_count = 0;
+    if (dr_doing_detach())
+        memset(&drwrap_stats, 0, sizeof(drwrap_stats_t));
```

Comment:
It seems good to put the postcall_cache array and probably the other stores above inside the same if() right?

Reply:
Sure. I just had inside the if() only stores with a little bit more cost, but I can move the rest there as well.